api/v1alpha1/gitopsservice_types.go

```diff
@@ -30,6 +30,8 @@ type GitopsServiceSpec struct {
 	RunOnInfra bool `json:"runOnInfra,omitempty"`
 	// Tolerations allow the pods to schedule onto nodes with matching taints
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+	// NodeSelector is a field of PodSpec, it is a map of key value pairs used for node selection
```

Comment:
I think this should give a little more clear explanation, like the one for `runOnInfra`. I.e. state that this applies to the default workloads of GitOps operator.

This will be shown to users upon `oc explain`, so indirectly is user documentation :)